src/renderer/containers/Console/AddConnectionForm.tsx

```diff
@@ -108,50 +114,79 @@ export const AddConnectionForm: React.FC<AddConnectionFormProps> = ({ defaultVal
+                </span>
+              }
+            />
+            <section>
+              <Collapse in={showAutoswitcher}>
+                <div
+                  css={css`
+                    display: grid;
+                    grid-gap: 10px;
+                    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+                    margin: 10px 0px 10px;
+                  `}
+                >
+                  <TextField
+                    label="OBS Websocket IP:Port"
+                    value={obsIP ?? ""}
+                    required={showAutoswitcher}
+                    onChange={(e) => setValue("obsIP", e.target.value)}
+                  />
+                  <TextField
+                    label="OBS Password"
+                    value={obsPassword ?? ""}
+                    onChange={(e) => setValue("obsPassword", e.target.value)}
+                    type="password"
+                  />
+                </div>
+                <TextField
+                  label="OBS Source Name"
+                  value={obsSourceName ?? ""}
+                  required={showAutoswitcher}
+                  onChange={(e) => setValue("obsSourceName", e.target.value)}
+                />
+              </Collapse>
+            </section>
+            <section>
+              <Toggle
+                value={Boolean(enableRelay)}
+                onChange={(checked) => setValue("enableRelay", checked)}
+                label="Console Relay"
+                description="The relay allows external programs (e.g. stream layouts) to tap into the raw Slippi data stream without affecting mirroring.
+                This connection's relay port will be shown on the console card after you have saved and is activated once you select connect."
               />
-              <TextField
-                label="OBS Password"
-                value={obsPassword ?? ""}
-                onChange={(e) => setValue("obsPassword", e.target.value)}
-                type="password"
+            </section>
+            <section>
+              <Toggle
+                value={showConnPort}
+                onChange={() => setShowConnPort(!showConnPort)}
+                label="Connection Port"
+                description="Set the port to connect to for mirroring. This is typically used with Console Relay from another computer to send mirror again"
               />
-            </div>
-            <TextField
-              label="OBS Source Name"
-              value={obsSourceName ?? ""}
-              required={showAdvanced}
-              onChange={(e) => setValue("obsSourceName", e.target.value)}
-            />
+              <Collapse in={showConnPort}>
+                <TextField
+                  label="Port Number"
+                  value={port ?? ""}
+                  required={showConnPort}
+                  onChange={(e) => setValue("port", Number(e.target.value.replace(/[^0-9]/g, "")))}
```

Comment:
This kinda replacement, where you convert numbers into text make the typing experience kinda wack cuz you cannot backspace to clear the field, it replaces it with zero. I would probably keep it as a string and maybe set it as a number on submit or something?

```suggestion
                  type="number"
                  onChange={(e) => setValue("port", e.target.value)}
```